modules/ocf_irc/templates/thelounge-conf.js.erb

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  public: true,
+  host: '127.0.0.1',
+  port: 9000,
+  bind: '127.0.0.1',
+  theme: "themes/example.css",
+  displayNetwork: false,
+  lockNetwork: true,
+
+  defaults: {
+    name: "OCF IRC",
+    host: "irc.ocf.berkeley.edu",
+    port: 6697,
+    tls: true,
+    nick: "",
+    username: "",
+    realname: "",
+    join: "#rebuild"
+  },
+}
```

Comment:
Oh, it originally had SSL certificates referenced in it that needed the hostname, but I removed that since it wouldn't have a HTTP -> HTTPS redirect. This shouldn't be a template now, thanks for catching that!